### PR TITLE
Update calculator.py

### PR DESCRIPTION
**Pull request by gideonbusayo**
Please see this review, and you may contact me for references. slack: @Busganda.
You've done a great job here, I just had to modify it. Best of luck.
See you around

### CODE REVIEW

File: calculator.py [hunk 1]
```diff
@@ -1,67 +1,79 @@
...
+      if calc_again.upper() == 'N':
+        end()
+      break
+    
 
-Welcome()
-Arithemetic_operations()
-runAgain()
-End()
+if __name__ == "__main__":
```
Comment (Wastino-zz) [inline calculator.py:78]: Why are we using a while loop here instead of a simple If/else. Any added benefit to the program?

Comment (Wastino-zz) [inline calculator.py:78]: _name_ is not defined, what is this line of code supposed to do

File: calculator.py [hunk 2]
```diff
@@ -1,67 +1,79 @@
-def Welcome():
+def main():
```
Comment (Wastino-zz) [inline calculator.py:1]: Is it best practice to call functions before the program? or at the end.

File: calculator.py [hunk 3]
```diff
@@ -1,67 +1,79 @@
...
-    elif Operation == '*':
-        print('{} * {} =' . format(Num_1, Num_2))
-        print(Num_1 * Num_2)
-    elif Operation == '**':
-        print('{} ** {} =' . format(Num_1, Num_2))
-        print(Num_1 ** Num_2)
-    elif Operation == "":
-        End()
-    else:
+    if operation != "0":
```
Comment (Wastino-zz) [inline calculator.py:24]: The rearrangement to this block of code is cool. Thanks gee

File: calculator.py (hunk 2 above)
Comment (gideonbusayo) [reply calculator.py:1]: At the end. Your code looks cleaner that way. You use function to break your code and make it simpler

File: calculator.py (hunk 1 above)
Comment (gideonbusayo) [reply calculator.py:78]: It makes it simpler. You using run_again() function inside its function complicates the code. I also realised that from your code, after pressing N, it still asks for user inputs then ends. I thought this was cooler.

**Review by Wastino-zz — APPROVED** [review]

**Pull request closed without merging**